make/jdk/src/classes/build/tools/taglet/SealedGraph.java

```diff
@@ -0,0 +1,259 @@
+/*
+ * Copyright (c) 2017, 2022, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.  Oracle designates this
+ * particular file as subject to the "Classpath" exception as provided
+ * by Oracle in the LICENSE file that accompanied this code.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+package build.tools.taglet;
+
+import com.sun.source.doctree.DocTree;
+import jdk.javadoc.doclet.Doclet;
+import jdk.javadoc.doclet.DocletEnvironment;
+import jdk.javadoc.doclet.Taglet;
+
+import javax.lang.model.element.*;
+import javax.lang.model.type.DeclaredType;
+
+import java.io.*;
+import java.util.*;
+import java.util.stream.Collectors;
+
+import static java.lang.System.lineSeparator;
+import static jdk.javadoc.doclet.Taglet.Location.TYPE;
+
+/**
+ * A block tag to optionally insert a reference to a sealed class hierarchy graph,
+ * and generate the corresponding dot file.
+ */
+public final class SealedGraph implements Taglet {
+    private static final String sealedGraphDotPath =
+            System.getProperty("sealedGraphDotPath");
```

Comment:
Generally, there is strong correlation between the module graph support and the sealed graph support, which is good, and helpful.   This system property is an outlier. For consistency with the world of module graphs, it should be `enableSealedGraph` or something like that.  Related: a change in the property name here will require a corresponding update in the makefiles.